add-on/panels/test/index.html

```diff
@@ -74,10 +76,12 @@
   <script src="roomViews_test.js"></script>
   <script src="l10n_test.js"></script>
   <script src="roomStore_test.js"></script>
+  <script src="slideshow_test.js"></script>
   <script>
     // Stop the default init functions running to avoid conflicts in tests
     document.removeEventListener('DOMContentLoaded', loop.panel.init);
     document.removeEventListener('DOMContentLoaded', loop.conversation.init);
+    document.removeEventListener("DOMContentLoaded", loop.slideshow.init);
```

Comment:
When running this locally, I was getting an error that `GetPluralRule` wasn't stubbed. It turns out that there's an intermittent failure as we're not removing the listener for the karma coverage. Please can you add this diff to the patch:

```diff
diff --git a/test/karma/tail.js b/test/karma/tail.js
index 0474ee4..f2807c3 100644
--- a/test/karma/tail.js
+++ b/test/karma/tail.js
@@ -12,6 +12,9 @@ if (loop) {
   if (loop.conversation) {
     document.removeEventListener("DOMContentLoaded", loop.conversation.init);
   }
+  if (loop.slideshow) {
+    document.removeEventListener("DOMContentLoaded", loop.slideshow.init);
+  }
 }
```